tests/Suite/Sdk/files/splitChanges.json

```diff
@@ -10,6 +10,7 @@
             "status": "ACTIVE",
             "killed": false,
             "defaultTreatment": "off",
+            "sets": [],
```

Comment:
Would things break if this field wasn't there?

Reply:
yes the Utilities that use the tests